cli/azd/cmd/cmd_help.go

```diff
@@ -82,7 +82,7 @@ func getCmdHelpDefaultDescription(cmd *cobra.Command) string {
 // getCmdHelpDefaultUsage provides the default implementation for displaying the help usage section.
 func getCmdHelpDefaultUsage(cmd *cobra.Command) string {
 	return fmt.Sprintf("%s\n  %s\n\n",
-		output.WithBold(output.WithUnderline("Usage")),
+		output.WithBold("%s", output.WithUnderline("Usage")),
```

Comment:
`WithBold` should really be called `WithBoldf` since it takes a format string.  Smarter linters were now complaining about passing non constant values, so we adopt this pattern.  I will do a pass later that adopts the `f` prefix pattern for all this stuff and make `WithBold` just take a single string argument and be sugar for `WithBoldf("%s", arg)`

Reply:
That makes a lot of sense!